SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/SubJob/SubJobWebRunner/BrokerProfileOptOutSubJobWebRunner.swift

```diff
@@ -141,7 +141,7 @@ public final class BrokerProfileOptOutSubJobWebRunner: SubJobWebRunning, BrokerP
 
     public func executeNextStep() async {
```

Comment:
Similar to scan sub jobs, while you’re at it, let’s add action log for “next action”, “releasing the web view”, “job completed”, “job cancelled”.